README.md

```diff
@@ -510,3 +514,43 @@ making it as generic as possible.
         with:
           install-command: npm install
 ```
+
+## Workflows
+
+### Tech docs push
+
+The workflow used to push the local documentation in any Piwik PRO repository to central Tech-docs portal. In order to use it, paste the following content into `.github/workflow/docs.yaml` in your repository:
+
+```yaml
+name: Documentation
+on: [push]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  push:
+
+    uses: PiwikPRO/actions/.github/workflows/push_docs.yaml@master
+    secrets: inherit
+    with:
+      config: |
```

Comment:
Maybe we should drop some hint about this:
```yaml
main_branch:
  default: master
  required: false
  type: string
```

We don't mention this in README, which could be valuable in some cases.